contracts/Exchange.sol

```diff
@@ -64,7 +64,7 @@ contract Exchange is Restricted {
 
     /* to allow upgrade of Rates  contract */
     function setRatesContract(Rates newRatesContract)
-    external restrict("MonetaryBoard") {
+    external restrict("StabilityBoardSignerContract") {
```

Comment:
Suggest renaming the permission name everywhere to just `StabilityBoard`, since this expresses the role clearly (and is shorter).